connbeat.full.yml

```diff
@@ -1,24 +1,609 @@
+################### Connbeat Configuration Example #########################
+
+# This file is an example configuration file highlighting only some common options.
+# The heartbeat.full.yml file in the same directory contains all the supported options
+# with detailed comments. You can use it for reference.
+#
+# You can find the full configuration reference here:
+# https://github.com/raboof/connbeat
+
+############################# Connbeat ######################################
+
 connbeat:
   expose_process_info: true
```

Comment:
Maybe you want to add docs on these values as well;

# bla bla
expose_process_info: true
..
..
# blah blah
#name: